src/StreamFactoryInterface.php

```diff
@@ -48,5 +16,5 @@ public function createStreamFromResource($body);
      *
      * @return StreamInterface
      */
-    public function createStreamFromString($body);
+    public function createStream($body = '');
```

Comment:
Probably it would be better to have a resource than string.

```php
interface StreamFactoryInterface
{    
    public function createStream($resource);
}
```

I am looking at  https://github.com/php-fig/http-message/blob/f6561bf28d520154e4b0ec72be95418abe6d9363/src/StreamInterface.php#L115 which already have a `write` method. So the implementation can make use of it.